app/views/candidate_mailer/unconditional_offer_accepted.text.erb

```diff
@@ -1,8 +1,8 @@
-Dear <%= @application_form.first_name %>,
+Dear <%= @application_form.first_name %>
 
-#You’ve accepted <%= @provider_name %>’s offer to study <%= @course_name_and_code %>
+You’ve accepted <%= @provider_name %>’s offer to study <%= @course_name_and_code %>.
 
-The course will start on <%= @start_date %>.
+The course starts <%= @start_date %>.
```

Comment:
> `The course starts October 2022.` 

Not sure that reads well 🤷🏼 (totally up to you though)

Reply:
They're all different, `@start_date` for this email only gives the Month and year

Reply:
maybe I'll keep as is then, because it works no matter the format (even if it doesn't read that well!)